docs/dc-apps-performance-toolkit-user-guide-confluence.md

```diff
@@ -306,6 +306,32 @@ For more information, go to [Administer your Data Center search index](https://c
     Snapshot was created successfully.
     ```
 
+## Execution environment
+
+There are two options for running DC App Performance Toolkit:
+ - Run Toolkit on your local machine.  It is recommended for spiking, debugging and custom actions developing. 
+ - Run Toolkit on the performer instance (dedicated AWS EC2 machine).  It is strongly recommended to use performer instance for submission process results generation.
+ In comparison with local execution running Toolkit on the performer instance provides fixed CPU, memory and network performance which guarantees stable and accurate results.
+ ### Local execution environment set up
+ Recommended hardware requirements:
+ - Supported platforms: Linux, MacOS, Windows
+ - CPU: 4 cores
+ - Memory: 16 GB
+
+To set up local execution environment follow [README.md](https://github.com/atlassian/dc-app-performance-toolkit) instructions.
+ 
+ ### Performer instance execution environment set up
+ 1. [Launch AWS EC2 instance](https://docs.aws.amazon.com/quickstarts/latest/vmlaunch/step-1-launch-instance.html). Instance type: [`c5.2xlarge`](https://aws.amazon.com/ec2/instance-types/c5/), OS: select from Quick Start `Ubuntu Server 18.04 LTS`.
```

Comment:
Check pls, for Confluence, we are using c5.4xlarge instance type

Reply:
@mmizin please be attentive and read one more time. It is NOT an instance for Confluence itself. This is performer instance requirements.